### DISCUSSION TITLE
[Mia-Platform Marketplace][Templates][Examples] Use of placeholder for files and folders names

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#293] [Ideas]
### Which area of this project could be most improved?

Marketplace

### Description

Hi guys,
a client asks us if it's possible to use, in marketplace templates/examples, _placeholders_ to be interpolated as file or folders names, e.g. I could have a file named `mia_template_service_name_placeholder`, like in the following example:
<img width="510" alt="Screenshot 2023-07-17 at 15 46 03" src="https://github.com/mia-platform/community/assets/28152586/06ef55fa-8a0b-4ea5-bb49-2cb4d4b64392">

The placeholders I am talking about are listed [here on the public documentation](https://docs.mia-platform.eu/docs/marketplace/add_to_marketplace/contributing_overview#configure-marketplace-strings).

I tried and, at the moment, it's not possible.
Despite that, is possible on "Projects templates", in which _mustache.js_ is used during the project creation.

What do you think about that?

**Comment by malta895** [comment]
Hi @silversoul93,

we can confirm you that at the moment we don't support variable interpolation in files and directory names.

However, thanks for the feedback, we may take this feature into consideration for the future developments of our roadmap @marcofilippi FYI

**Reply by frengooo** [reply]
I tried too by myself, and I can confirm it doesn't work 😢 

Thanks @malta895. @marcofilippi it's a very useful feature, a couple of clients/prospects asked me to add it.

**Comment by akelity** [comment]
This feature has been positively evaluated and may be considered for inclusion in the roadmap in the future, depending on priorities and overall specific needs.
